Packages/vcs/Lib/VTKPlots.py

```diff
@@ -87,7 +90,7 @@ def endEvent(self,obj,event):
     if self.renWin is not None:
       if self.reRender:
         self.reRender = False
-        self._lastSize = None
+        #self._lastSize = None
```

Comment:
This line was added in 72dcdc9238ce553ab18d3fc6b39487967e696a0b, but the commit message ("ok works on Mac") doesn't really  give us much to go on in terms of what was fixed...Hopefully @doutriaux1 remembers why this was needed ;)